My Submission for Mayhemheroes Bounty

mattjurenka opened the pull request.

mattjurenka committed:
initialized mayhem build

xansec commented on fuzz/Cargo.toml:
```diff
@@ -1,5 +1,3 @@
-cargo-features = ['named-profiles']
```
Can you comment on why these changes were necessary?

mattjurenka committed:
reworked fuzz cargo.toml profiles

mattjurenka commented:
Rust recently stabilized the named profiles feature, causing Cargo.toml to not parse on nightly. The debug profile must be renamed as well as it is now a reserved keyword. I fixed this originally by just removing the two empty profiles, but I just added a commit reversing this deletion to better match the original project structure.

xansec commented:
Makes sense, thanks!
